Source/WTF/wtf/text/StringView.h

```diff
@@ -68,11 +68,12 @@ class StringView final {
     StringView(const UChar*, unsigned length);
     StringView(const char*, unsigned length);
     StringView(ASCIILiteral);
-    ALWAYS_INLINE StringView(Span<const LChar> characters) : StringView(characters.data(), characters.size()) { }
-    ALWAYS_INLINE StringView(Span<const UChar> characters) : StringView(characters.data(), characters.size()) { }
 
     ALWAYS_INLINE static StringView fromLatin1(const char* characters) { return StringView { characters }; }
 
+    ALWAYS_INLINE static StringView fromSpan(Span<const LChar> span) { return StringView { span.data(), static_cast<unsigned>(span.size()) }; }
```

Comment:
It would be better to organize and comment these more like the `String` ones, which are clear on the character types and thus seem organized. And this one should be named `fromLatin1Span`.